### DISCUSSION TITLE
Installation on Oracle Cloud Infrastructure

### DISCUSSION THREAD

**Discussion by P1xelC0da** [kimai/kimai#4023] [On-Premise: Installation & Updates] [answered]
Hello, I was wondering if Kimai would run on an Oracle arm64 instance?

**Comment by kevinpapst** [comment]
Please share your experience after giving it a try.
I guess it mainly depends on availability of PHP and MySQL builds for it.

**Comment by P1xelC0da** [comment]
Will do! Guess I’m diving in on this head first.

---
Damian Freeman
Executive Director of Engineering, Chair of the Board of Directors
Devsaurus

**Comment by P1xelC0da** [comment, marked as answer]
Update: It works, and it runs flawlessly. Only issue I encountered when following the steps at https://www.kimai.org/documentation/fresh-ubuntu-20.html was the composer bit. Although I just found the proper steps on their site to go from `Installer corrupt` to `Installer verified` or whatever.

Thanks :D

**Reply by kevinpapst** [reply]
Fantastic. Thanks for sharing!

**Reply by P1xelC0da** [reply]
Is there a way I can change a username though? I fumbled and did my first name, versus staff ID.

**Reply by kevinpapst** [reply]
No. Best is to delete and re-create.

**Reply by P1xelC0da** [reply]
Thanks, just follow the same steps on the site?

**Reply by kevinpapst** [reply]
If you are able to install a server system, you are able to use the software. I trust your instinct and skillset!

**Reply by P1xelC0da** [reply]
Thank you for the kind words. :D It means a lot.

**Reply by P1xelC0da** [reply]
OK, well I rebooted my system and now it doesn't seem to start.

**Reply by kevinpapst** [reply]
I doubt that this is a Kimai issue

**Reply by P1xelC0da** [reply]
it wasn't, seems like an iptables conflict. now i'm getting a 500 internal server error @ https://time.devsaurus.xyz

**Reply by P1xelC0da** [reply]
Alright, the fact that I'm having issues with simple things is ludicrous lol. But I assume the reason I can see Kimai at https://devsaurus.xyz and other places with A Records is because I have nothing pointing to them in my nginx configs?

**Reply by kevinpapst** [reply]
You know your setup better than I do 😁 So all I can say is: maybe 🤷

**Reply by P1xelC0da** [reply]
Yess!! So within these two hours I've deployed kimai, set it up, debugged issues, and am now understanding the workings of nginx!! Thanks so much!!
